bmorph/core/bmorph.py

```diff
@@ -66,10 +66,10 @@ def edcdfm(raw_x, raw_cdf, train_cdf, truth_cdf):
            for x in raw_x]
 
     # Given u_t and train_cdf determine train_x
-    train_x = scipy.stats.scoreatpercentile(train_cdf, u_t)
+    train_x = pd.np.percentile(train_cdf, u_t)
 
     # Given u_t and truth_cdf determine truth_x
-    truth_x = scipy.stats.scoreatpercentile(truth_cdf, u_t)
+    truth_x = pd.np.percentile(truth_cdf, u_t)
```

Comment:
just a note for future reference, you can drop the `pd` (pandas) from these function calls. You are actually use numpy here.

Reply:
I know but I would need to add `import numpy as np` at the top which is not there currently (and I was lazy)